src/helper/blob-tools/blob.js

```diff
@@ -259,17 +263,15 @@ class Blobbiness {
             clearSelection(this.clearSelectedItems);
             items = getItems({
                 match: function (item) {
-                    return blob.isMergeable(lastPath, item) && blob.touches(lastPath, item);
+                    return blob.isMergeable(lastPath, item) &&
+                        blob.touches(lastPath, item) &&
+                        !isCompoundPathChild(item);
                 },
                 class: paper.PathItem
             });
         }
 
         for (let i = items.length - 1; i >= 0; i--) {
-            // If a path is part of a compound path, that parent path will later be processed.
-            // Skip processing the child path so as not to double-process it.
-            if (isCompoundPathChild(items[i])) continue;
-
             // TODO handle compound paths
```

Comment:
Maybe that's what this comment was referring to? =_=;

Reply:
I was wondering about that... is it okay to remove those comments then?

Reply:
We're handling them now, so I guess so!